src/lib/__tests__/Employee.test.js

```diff
@@ -0,0 +1,36 @@
+// using Employee constructor
+const Employee = require("../../lib/Employee");
+
+describe("Employee", () => {
+  test("should be an instance of Employee", () => {
+    const expected = "Employee";
+    const instance = new Employee("Employee");
```

Comment:
Instead of doing this repeatedly in each test block, we could generate a complete instance of an employee in the `describe` block:

```
const instance - new Employee( { name: "Conor Kelly", id: "1234551241", email: "conorjkelly96@gmail.com" })
```

And then use this in each of our test blocks to check against.